apps/state/lib/state/server.ex

```diff
@@ -118,7 +119,7 @@ defmodule State.Server do
       Indicies in the `:mnesia` table where state is stored
       """
       @spec indicies :: [atom]
-      def indicies, do: unquote(opts[:indicies])
+      def indicies, do: unquote(indicies)
```

Comment:
I believe the correct spelling is "indices". But I see this is a configurable option and a public function so changing it might have far reaching implications that might not be worth it. Could always live with it a la HTTP Referer.

Reply:
I'll put in a separate ticket to address.